DQMOffline/ParticleFlow/python/runBasic_step2_cfg.py

```diff
@@ -0,0 +1,50 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process('HARVESTING')
+
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.load('Configuration.EventContent.EventContent_cff')
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
+process.load('Configuration.StandardSequences.MagneticField_AutoFromDBCurrent_cff')
+process.load('Configuration.StandardSequences.EDMtoMEAtRunEnd_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+# load DQM
+process.load("DQMServices.Core.DQM_cfg")
+process.load("DQMServices.Components.DQMEnvironment_cfi")
+
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(1)
+)
+
+# Input source
+process.source = cms.Source("DQMRootSource",
+                            fileNames = cms.untracked.vstring("file:OUT_step1.root"))
+#process.source = cms.Source("DQMRootSource",
+#                            fileNames = cms.untracked.vstring("file:test.root"))
+
+#process.source = cms.Source("PoolSource", fileNames = cms.untracked.vstring())
```

Comment:
If the alternative PoolSource will not be used, consider removing it to avoid confusion.